### PR TITLE
Update HC guide with oVirt 4.2 deployment and features

### PR DESCRIPTION
**Pull request by sabose**
Signed-off-by: Sahina Bose <sabose@redhat.com>
Changes proposed in this pull request:

- Updated the guide with content relevant for 4.2
- Added details on the fields used in the wizard in response to user queries

- confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sabose

This pull request needs review by: @rohantmp @gobindadas @ezio-auditore @akashihi

### CODE REVIEW

**Comment by gregsheremeta** [comment]
@sabose we're in the process of updating all docs from 4.0 to 4.2. When that is ready, I think it would overwrite this. Thoughts?

**Comment by sabose** [comment]
@gregsheremeta is there a planned update to HC guides in the 4.2 doc refresh? If not, I would still go with merging this, and relook at content post the 4.2 doc update

**Comment by rohantmp** [comment]
Looks good, +1

File: source/documentation/gluster-hyperconverged/chap-Deploying_Hyperconverged.html.md [hunk 1]
```diff
@@ -46,7 +46,71 @@ Refer [oVirt Nodes](../install-guide/chap-oVirt_Nodes) for instructions on insta
...
+
+More details on the deployment wizard are explained below
+
+#### Host selection
+
+To ensure that GlusterFS related network traffic is segregated on the backend/storage network, the gluster bricks need to be defined using these interface. In this tab, provide the FQDN/IP addresses associated with the backend/storage network so that the bricks are correctly defined during the deployment process.
+
+Cockpit wizard uses ansible to perform the gluster deployment on the hosts. It is required to have key-based authentication setup for `root` user between the host running the wizard and the 3 addresses providedfor gluster hosts
+
+        On the host from which deployment is run
```
Comment (akashihi) [inline source/documentation/gluster-hyperconverged/chap-Deploying_Hyperconverged.html.md]: You should mention, that it is a first host in a host list, not a host, where wizard is running

Comment (sabose) [reply source/documentation/gluster-hyperconverged/chap-Deploying_Hyperconverged.html.md]: Done

**Comment by sabose** [comment]
Any further comments, or can this be merged?

**Comment by gregsheremeta** [comment]
> Any further comments, or can this be merged?

lgtm, merging

[ @sabose, @shaunix will be doing the 4.2 HC Guide conversion in March when he has time, so we'll need to compare content then ]

**Pull request merged**